Catappult/appcoins-wallet-android
# [APPC-4280] Settings update community section

**What does this PR do?**

   Tasks: 
      1. Replace the Telegram entry with a Discord entry.
      2. Replace the 'X' icon.

**Database changed?**

   No

**Where should the reviewer start?**

   Check the SettingsScreen and validate that the Telegram button was replaced with the Discord (click discord to validate that it opens Discord). Also, validate that the X icon was replaced with the new one.

**How should this be manually tested?**

   Check the SettingsScreen and validate that the Telegram button was replaced with the Discord (click discord to validate that it opens Discord). Also, validate that the X icon was replaced with the new one.

**What are the relevant tickets?**

  Tickets related to this pull-request: [APPC-4280](https://aptoide.atlassian.net/browse/APPC-4280)

**Code Review Checklist**



[APPC-4280]: https://aptoide.atlassian.net/browse/APPC-4280?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
